client/log_verifier.go

```diff
@@ -40,6 +41,27 @@ func NewLogVerifier(hasher hashers.LogHasher, pubKey crypto.PublicKey) LogVerifi
 	}
 }
 
+// NewLogVerifierFromTree creates a new LogVerifier
+func NewLogVerifierFromTree(config *trillian.Tree) (LogVerifier, error) {
+	if got, want := config.TreeType, trillian.TreeType_LOG; got != want {
+		return nil, fmt.Errorf("client: NewFromTree(): TreeType: %v, want %v", got, want)
```

Comment:
errors still reference NewFromTree() here and the two below.

Reply:
done